docs/content/config.md

```diff
@@ -4,61 +4,61 @@ description: "use workspacer like a pro!"
 type: faq
 ---
 
-Workspacer is configured with the C# programming language. The expressiveness of C# allows you to be hyper-specifc in terms of describing your desired behavior.
-This configuration documentation assumes that you have already created the example config in the correct folder, if you haven't, check the [quickstart guide](/quickstart).
+Workspacer is configured with the C# programming language. The expressiveness of C# allows you to be hyper-specific in terms of describing your desired behavior.
```

Comment:
behaviour

Reply:
"behavior" is the preferred spelling in American English, which is what I assumed we'd go with as Rick lives in NYC

Reply:
You're right, I'm studying british english and I forgot we write things differently